ansible/roles/biocache-service/tasks/main.yml

```diff
@@ -268,14 +268,14 @@
   tags:
     - biocache-service  
 
-- name: copy all data assets
-  copy: src=data/config dest={{data_dir}}/biocache owner={{tomcat_user}}  group={{tomcat_user}}
-  tags:
-    - biocache-service
-    - properties
-
 - name: copy all config.properties
-  template: src=config/download-doi-email.html dest={{data_dir}}/biocache/config/
+  template: src={{ item }} dest={{data_dir}}/biocache/config/{{ item | basename | regex_replace('\.j2','') }}
```

Comment:
Jinja2 and ansible will recognise the templates even without the `.j2` extension. I will merge this manually and remove the extensions to simplify this line. Thanks!

Reply:
Sounds great. thank you.